stacks-signer/src/client/stackerdb.rs

```diff
@@ -208,15 +205,18 @@ impl<M: MessageSlotID + 'static> StackerDB<M> {
                     Some(StackerDBErrorCodes::DataAlreadyExists) => {
                         if let Some(slot_metadata) = chunk_ack.metadata {
                             warn!("Failed to send message to stackerdb due to wrong version number. Attempted {}. Expected {}. Retrying...", slot_version, slot_metadata.slot_version);
-                            slot_version = slot_metadata.slot_version;
+                            self.signer_db
+                                .set_latest_chunk_version(
+                                    &signer_pk,
+                                    slot_id.0,
+                                    slot_metadata.slot_version,
+                                )
+                                .map_err(ClientError::SignerDBError)?;
```

Comment:
```suggestion
                                )?;
```

Reply:
Changed in https://github.com/stacks-network/stacks-core/pull/6106/commits/18b0f07b37e8cab1f36bcb11520525fd42ec6bf0